mediapipe/calculators/ovms/openvinoinferencecalculator.cc

```diff
@@ -48,32 +48,6 @@
 namespace mediapipe {
 using std::endl;
 
-namespace {
-
-#define ASSERT_CIRCULAR_ERR(C_API_CALL) \
-    {                                   \
-        auto* fatalErr = C_API_CALL;    \
-        RET_CHECK(fatalErr == nullptr); \
-    }
-
-#define ASSERT_CAPI_STATUS_NULL(C_API_CALL)                                                 \
-    {                                                                                       \
-        auto* err = C_API_CALL;                                                             \
-        if (err != nullptr) {                                                               \
-            uint32_t code = 0;                                                              \
-            const char* msg = nullptr;                                                      \
-            ASSERT_CIRCULAR_ERR(OVMS_StatusCode(err, &code));                               \
-            ASSERT_CIRCULAR_ERR(OVMS_StatusDetails(err, &msg));                             \
-            LOG(INFO) << "Error encountred in OVMSCalculator:" << msg << " code: " << code; \
-            OVMS_StatusDelete(err);                                                         \
-            RET_CHECK(err == nullptr);                                                      \
-        }                                                                                   \
-    }
-#define CREATE_GUARD(GUARD_NAME, CAPI_TYPE, CAPI_PTR) \
-    std::unique_ptr<CAPI_TYPE, decltype(&(CAPI_TYPE##Delete))> GUARD_NAME(CAPI_PTR, &(CAPI_TYPE##Delete));
-
-}  // namespace
-
```

Comment:
Don't we need it anymore?

Reply:
It looks like leftover from time when we did not use Adapters.

Reply:
Not needed in this file.